condor/migration-guide.md

```diff
@@ -0,0 +1,193 @@
+# Casper 2.0 Migration Guide for Crypto Exchanges
+
+The Casper Network is in the process of upgrading to Casper 2.0, which introduces breaking changes to the node software, the protocol and the interfaces associated with interacting with the network. In order to maintain compatibility with Casper Network after the upgrade of mainnet, tentatively scheduled for March 2025, crypto exchanges that offer $CSPR pairs, allow for $CSPR deposits and withdrawals and/or allow their users to stake $CSPR, may need to upgrade their integration with Casper Network. This migration guide will help you determine whether, and if so - how, to migrate your integration in prepartion for the network upgrade.
```

Comment:
Let's shorten `the interfaces associated with interacting with the network` to `the API interfaces`

Reply:
Technically, there should be an Oxford comma after `the protocol` in `the node software, the protocol, and the interfaces`, but I don't want to start a fight here. Let's just make sure we use the oxford comma consistently.